### PR TITLE
Minor improvements to man pages

### PR DESCRIPTION
**Pull request by dtardon**
Mostly markup-related changes.

### CODE REVIEW

File: man/veritytab.xml [hunk 1]
```diff
@@ -41,17 +41,17 @@ This is based on crypttab(5).
     verity protected block device. Fields are delimited by
     white space.</para>
 
-    <para>Each line is in the form<programlisting><replaceable>volume-name</replaceable> <replaceable>data-device</replaceable> <replaceable>hash-device</replaceable> <replaceable>roothash</replaceable> <replaceable>options</replaceable></programlisting>
+  <para>Each line is in the form<programlisting><replaceable>volume-name</replaceable> <replaceable>data-device</replaceable> <replaceable>hash-device</replaceable> <replaceable>roothash</replaceable> <optional><replaceable>options</replaceable></optional></programlisting>
```
Comment (yuwata) [inline man/veritytab.xml]: indentation?

**Review by yuwata — APPROVED** [review]
LGTM, but one nit.

**Comment by yuwata** [comment]
Please rebase.

**Comment by dtardon** [comment]
I've retracted one commit ("man: simplify xpointer expression"), because it turns out that the new expression can fail if DTD failed to load. That's because the `id()` function only works on attributes of type `ID` and the type is defined in DTD.

But I've added one fix as a compensation :-)

File: man/veritytab.xml (hunk 1 above)
Comment (dtardon) [reply man/veritytab.xml]: Fixed.

**Pull request merged**